packaging/scripts/lib/assemble.ps1

```
@@ -0,0 +1,236 @@
+    [string]$SourceDir  = './source',
+    [string]$OutDir     = './out',
+    [Parameter(Mandatory)][string]$SourceCommit,
+    [Parameter(Mandatory)][string]$Version,
+    [Parameter(Mandatory)][string]$CmidPseudoVersion,
+    [switch]$AllowUnsigned
+)
+
+Set-StrictMode -Version Latest
+$ErrorActionPreference = 'Stop'
+
+function DefenseClaw-Die {
```

PSScriptAnalyzer (1.25.0)

[warning] The cmdlet 'DefenseClaw-Die' uses an unapproved verb.

(PSUseApprovedVerbs)